### ISSUE TITLE
The parameter settings for MobileNet V2.

### ISSUE THREAD

**Issue by eliac1988** [taehoonlee/tensornets#23]
Hi,
I would like to ask about the parameter settings of training MobileNet V2 from scratch with ImageNet, such as the initial learning rate, the optimizer, the loss function, as well as the training epochs. 

Thank you so much.

**Comment by taehoonlee** [comment]
@eliac1988, This repository is initially designed for only inference. I'm planning to add training codes near soon, but currently, you can refer to the original TF-slim repository:

- [`train_image_classifier` script](https://github.com/tensorflow/models/blob/master/research/slim/train_image_classifier.py)
- [hyper-parameters for training](https://github.com/tensorflow/models/tree/master/research/slim/nets/mobilenet#training)
